.kitchen.docker.yml

```diff
@@ -0,0 +1,77 @@
+---
+driver:
+  name: docker
+
+provisioner:
+  name: puppet_apply
+  manifests_path: environments/etc/manifests
+  modules_path: /tmp/modules
+  require_puppet_repo: true
+  require_chef_for_busser: false
+  puppet_debug: true
+  puppet_verbose: true
+  custom_pre_apply_command: 'cp -r /tmp/modules/* /tmp/kitchen/modules/'
+
+platforms:
+  - name: centos-7
+    driver_config:
+      require_chef_omnibus: 14.12
+      image: 'datadog/docker-library:chef_kitchen_systemd_centos_7'
+      run_command: /root/start.sh
+
+    driver:
+      provision_command:
+        - rpm -ivh http://yum.puppetlabs.com/puppetlabs-release-el-7.noarch.rpm
+        - yum install -y puppet
+
+        - mkdir /home/kitchen/puppet
+        - printf <%= File.read('environments/etc/Puppetfile').inspect %> > /home/kitchen/puppet/Puppetfile
+
+        - gem install r10k -v 2.6.7
+        - cd /home/kitchen/puppet && r10k puppetfile install --moduledir=/tmp/modules
+
+  - name: centos-8
+    driver_config:
+      require_chef_omnibus: 14.12
+      image: 'datadog/docker-library:chef_kitchen_systemd_centos_8'
+      run_command: /root/start.sh
+
+    driver:
+      provision_command:
+        - dnf install -y https://yum.puppetlabs.com/puppet-release-el-8.noarch.rpm
+        - dnf install -y puppet
+
+        - mkdir /home/kitchen/puppet
+        - printf <%= File.read('environments/etc/Puppetfile').inspect %> > /home/kitchen/puppet/Puppetfile
+
+        - gem install r10k -v 2.6.7
+        - cd /home/kitchen/puppet && r10k puppetfile install --moduledir=/tmp/modules
+
+  - name: ubuntu-16.04
+    driver:
```

Comment:
Should we add `driver_config` here as well? Maybe ubuntu 16.05 is the default, but that's not obvious and could change (also it's more consistent).

Reply:
Yeah, I can fix an image but this should normally use the official one